tests/integration/api/queries/ordersByAccountId/ordersByAccountId.test.js

```diff
@@ -0,0 +1,233 @@
+import Factory from "/imports/test-utils/helpers/factory";
+import TestApp from "/imports/test-utils/helpers/TestApp";
+
+jest.setTimeout(300000);
+
+const shopId = "integ-test-shop-id";
+const accountId = "integ-test-account-id";
+const opaqueAccountId = "cmVhY3Rpb24vYWNjb3VudDppbnRlZy10ZXN0LWFjY291bnQtaWQ=";
+const orderId = "integ-test-order-id";
+const mockAccount = Factory.Account.makeOne({
+  _id: accountId
+});
+
+const ordersByAccountIdQuery = `query ($accountId: ID!, $orderStatus: [String]) {
```

Comment:
I think this query should be separated into a [separate graphql file](https://github.com/reactioncommerce/reaction/blob/master/tests/integration/api/queries/account/AccountFullQuery.graphql) like it is in [accounts.](https://github.com/reactioncommerce/reaction/blob/master/tests/integration/api/queries/account/account.test.js#L3) I think it simplifies this file and allows for easier readability since this query is _so long_. The shorter queries that are 2 lines don't need separation, but I think it would cleaner to separate this into a graphql file. Hope this helps :)